packages/http/src/mocker/errors.ts

```diff
@@ -18,6 +18,12 @@ export const NOT_FOUND: Omit<ProblemJson, 'detail'> = {
   status: 404,
 };
 
+export const NO_SUCCESS_RESPONSE_DEFINED: Omit<ProblemJson, 'detail'> = {
+  type: 'NO_SUCCESS_RESPONSE_DEFINED',
+  title: 'No 2** response defined, cannot mock',
```

Comment:
Maybe "No response with code 200-299 defined..."?

While we totally understand the "2** response" version, I'm afraid it might not be as clear for users less experienced in the domain.

Reply:
@marcelltoth "No response in the range 200-299 defined" ?

Reply:
I'm cool with that.